api/routes/itemRouter.js

```diff
@@ -0,0 +1,8 @@
+const express  = require('express');
+const { itemController } = require('../controllers');
+const router = express.Router();
+
+router.get('/all', itemController.getAllItems);
+router.get('/item', itemController.getItems);
+router.get('/new', itemController.getItemMain);
```

Comment:
사실 main에서 사용하는 item 목록도 getItems를 사용하면 제일 좋지만 일단은 다음과 같이 라우터를 수정하면 좋을 것 같습니다.
```suggestion
router.get('/mian', itemController.getItemMain);
```